android/app/src/main/java/com/sample/react/TestAppReactNativeHost.kt

```diff
@@ -12,8 +15,8 @@ import javax.inject.Singleton
 
 @Singleton
 class TestAppReactNativeHost @Inject constructor(
-        application: Application,
```

Comment:
Android Studio added the extra indentation. Do you have different setting or something?

Reply:
I noticed the difference too, and double checked the code style. Mine is set to the Kotlin Guidelines code style and it formats files as you see in this PR. Can you check what happens on your side?

Reply:
Continuation indent is set to 8. I haven't changed this so I'm assuming it's default.
<img width="1126" alt="image" src="https://user-images.githubusercontent.com/4123478/79559503-27dddf80-80a6-11ea-8729-111b9b1ca8e9.png">

Reply:
Can you click on `Set From -> Predefined style -> Kotlin style guide`?

Reply:
No diff

Reply:
Crap.. Which version of AS?

Reply:
3.6.2

Reply:
I am running on 3.6.1, let me updated and check the new version.

Reply:
Updated, no diff. Reset to defaults, and then imported the kotlin style, no diff :D

Reply:
Can you close AS, remove `~/Library/Preferences/AndroidStudio3.6`, then try again?

Reply:
Well, that bricked my AS installation :D

Reply:
After re-installing the AS, **default** code style was changing formatting similar to what you had, but if you set the kotlin style, then it will leave it as it is.

Reply:
You should try first to reset the style to default, and then apply the kotlin style guide.

Reply:
Can we please use whatever comes out of box? I don't want all contributors to have to do this.

Reply:
Well, this *is*, in fact, is the android style guide: https://developer.android.com/kotlin/style-guide

Reply:
For some reason, it is not set by default in AS, even though they ship the style.

Reply:
Copy/pasting the comment from the other PR: Can we check in the settings so we don't have to ask contributors to change their settings in AS?

Reply:
Ah, wait, crap, they've switched it around sometime ago and I haven't noticed. Now the style guide in the settings is this one: https://kotlinlang.org/docs/reference/coding-conventions.html. Oops..